apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/InMemoryNormalizedCacheStore.java

```diff
@@ -0,0 +1,27 @@
+package com.apollographql.android.cache.normalized;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+public class InMemoryNormalizedCacheStore extends NormalizedCacheStore {
+
+  private final Map<String, ApolloRecord> store = new HashMap<>();
```

Comment:
nitpick and sorta opinion to use LinkedHashMap https://publicobject.com/2016/02/08/linkedhashmap-is-always-better-than-hashmap/  also do we need to worry about threading here?